deployment/deploy_api_docs.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+TAG_MAIN=`git describe --tags`
+
+git clone git@github.com:aeternity/epoch-api-docs.git /tmp/epoch-api-docs
+cp apps/aehttp/priv/swagger-docs/swagger.json /tmp/epoch-api-docs/
+
+cd /tmp/epoch-api-docs/
+
+git add swagger.json;
+git config user.email "circleci@aeternity.com";
+git config user.name "CircleCi Aeternity";
+if [ -z `git status --porcelain` ]; then
+    echo "Nothing to commit";
+else
+    git commit -a -m 'Update epoch docs CircleCI';
+    git push origin master;
+    git tag $TAG_MAIN
```

Comment:
As far as I can understand this it retag latest tag with the latest commit ?

I think you should use and check `CIRCLE_TAG` from https://circleci.com/docs/2.0/env-vars/#build-details
Somethign like:
```
if $CIRCLE_TAG
git tag $CIRCLE_TAG
git push origin $TAG_MAIN
else
git commit -a -m 'Update epoch docs CircleCI'
git push origin master
fi

Reply:
I'm not still sure what should be pushed to the docs master.
1. epoch master docs
2. epoch lastest release docs

@sennui @lucafavatella What do you think?

Reply:
yes, CIRCLE_TAG will be better option here. Will fix. also yes, when and what should be pushed to docs :D and when ;]

Reply:
Given that we have single docs repo that gets hosted as official doc it has to be the latest official release.

I wonder how we can help out our aepps team that would like to use newer version that is not yet released. API repo per environment & push to API repo during deployment work?

Reply:
@dincho I would expect master to contain the docs generated from epoch master.

Though I would expect the docs of each release to be pushed as well - with tag (or branch - whatever easier) name inferred from epoch tag names.

Reply:
@lucafavatella unfortunately only one version is pretty printed

Reply:
@dincho updatet to user CIRCLE_TAG if exists to make commit tag to api-docs repo

Reply:
@michalzajda we can hack on this with some 3td party tools like:
- https://htmlpreview.github.io
- https://rawgit.com